### PR TITLE
Feature/modal window

### PR DESCRIPTION
**Pull request by Polkan37**
Add modal window with form for order

### CODE REVIEW

File: src/components/PriceSection/PriceSection.js [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
+import React, {useState} from 'react'
 import "./price.css"
-
 let cardInfo = [
     { id: 1, name: 'Минимум', price: '100$', options: ['Съемка - 60 минут','Обратботка -  14 дней','Обработка 50 удачных снимков','Консультация по подбору образа'] },
     { id: 2, name: 'Стандарт', price: '200$', options: ['Съемка - 2 часа','Обратботка -  21 день','Обработка 100 удачных снимков','Консультация по подбору образа'] },
     { id: 3, name: 'Премиум', price: '150$/час', options: ['Мин. время - 4 часа','Обратботка -  до 50 дней','Ретушь крупных планов','Обработка всех удачных снимков', 'Фотокнига из 20 кадров'] },
 ];
-const selectCard = (deg = 0, info = cardInfo[1]) => {
+const cards = ['min', 'standart', 'max'];
```
Comment (Boromir1994) [inline src/components/PriceSection/PriceSection.js:8]: move to constants

File: src/components/PriceSection/PriceSection.js [hunk 2]
```diff
@@ -82,9 +86,9 @@ function PriceSection() {
         <div className="mobile-price__container container">
           
           <div className="mobile-price__titles">
-            <div className="card__title" onClick={() => {selectCard(-110, cardInfo[0])}}>{cardInfo[0].name}</div>
-            <div className="card__title selected" onClick={() => {selectCard(0, cardInfo[1])}}>{cardInfo[1].name}</div>
-            <div className="card__title" onClick={() => {selectCard(100, cardInfo[2])}}>{cardInfo[2].name}</div>
+            <div className="card__title" id="min" onClick={() => {selectCard(-100, 'min', cardInfo[0])}}>{cardInfo[0].name}</div>
+            <div className="card__title selected" id="standart" onClick={() => {selectCard(0, 'standart', cardInfo[1])}}>{cardInfo[1].name}</div>
+            <div className="card__title" id="max" onClick={() => {selectCard(110, 'max', cardInfo[2])}}>{cardInfo[2].name}</div>
```
Comment (Boromir1994) [inline src/components/PriceSection/PriceSection.js:91]: -100 and 100 should be in constants

File: src/components/PriceSection/PriceSection.js (hunk 1 above)
Comment (Polkan37) [reply src/components/PriceSection/PriceSection.js:8]: I will fix it in new branch

File: src/components/PriceSection/PriceSection.js (hunk 2 above)
Comment (Polkan37) [reply src/components/PriceSection/PriceSection.js:91]: I will fix it in new branch

**Pull request merged**